### DISCUSSION TITLE
Automatically create issue in linked versioning system.

### DISCUSSION THREAD

**Discussion by mikebronner** [spatie/flareapp.io-roadmap#100] [Ideas]
It would be very helpful to automatically create linked issues in linked versioning systems, like Github, Bitbucket, etc. This would improve transparency for devs who don't have direct Flare access, as well as allow easy creation of PRs based on tickets, and not rely on a person to manually do something that could be automated.

In our team we want to eliminate as much tedium as we can, so that devs can focus on dev'ing, instead of issue management, etc. I think this would be a welcome addition for anyone using Flare.

Thanks!

**Comment by glennjacobs** [comment]
We would also like this functionality, specifically for Linear, so that new Flare issues are created in the triage section.
